docs/how-to/understanding-omnitrace-output.rst

```diff
@@ -311,16 +311,17 @@ set ``OMNITRACE_OUTPUT_PREFIX="%argt%-"``, and let Omnitrace cleanly organize th
 .. note::
 
    In any output prefix key which contains a ``/`` character, the ``/`` characters
-   are replaced with ``_`` and any leading underscores are stripped. For example, 
-   an ``%arg0%`` of ``/usr/bin/foo`` translates to ``usr_bin_foo``. Additionally, any ``%arg<N>%`` keys which 
+   are replaced with ``_`` and any leading underscores are stripped. For example,
+   an ``%arg0%`` of ``/usr/bin/foo`` translates to ``usr_bin_foo``. Additionally, any ``%arg<N>%`` keys which
    do not have a command line argument at position ``<N>`` are ignored.
 
 Perfetto output
 ========================================
 
-Use the ``OMNITRACE_OUTPUT_FILE`` to specify a specific location. If this is an 
+Use the ``OMNITRACE_OUTPUT_FILE`` to specify a specific location. If this is an
 absolute path, then all ``OMNITRACE_OUTPUT_PATH`` and similar
-settings are ignored. Visit `ui.perfetto.dev <https://ui.perfetto.dev>`_ and open this file.
+settings are ignored. Visit `ui.perfetto.dev <https://ui.perfetto.dev>`_ and open
+this file (See `NOTE <https://github.com/ROCm/omnitrace/blob/main/README.md>`_).
```

Comment:
```suggestion
this file.

.. important::

   Perfetto validation is done with trace_processor v46.0 as there is a known issue with v47.0.
   If you are experiencing problems viewing your trace in the latest version of `Perfetto <http://ui.perfetto.dev>`_,
   then try using `Perfetto UI v46.0 <https://ui.perfetto.dev/v46.0-35b3d9845/#!/>`_.
```

IMO, "See NOTE" is kind of confusing IMO. Maybe we could just directly inform the user about the issue.

![image](https://github.com/user-attachments/assets/2b450c26-4f80-4ead-a60b-90aa9da4a3f8)